fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchNumberTest.java

```diff
@@ -193,8 +193,8 @@ public void testSearchNumber_decimal() throws Exception {
         assertSearchDoesntReturnSavedResource("decimal", "lt99");
         assertSearchDoesntReturnSavedResource("decimal", "lt99.98499");
         assertSearchDoesntReturnSavedResource("decimal", "lt99.985");
-        assertSearchDoesntReturnSavedResource("decimal", "lt99.99");
-        assertSearchDoesntReturnSavedResource("decimal", "lt9999e-2");
```

Comment:
I'm almost positive we've discussed this... this one is checking range? if so, this makes sense